ml-dsa/src/lib.rs

```diff
@@ -156,6 +191,23 @@ impl<P: MlDsaParams> signature::KeypairRef for KeyPair<P> {
     type VerifyingKey = VerifyingKey<P>;
 }
 
+impl<P> TryFrom<PrivateKeyInfoRef<'_>> for KeyPair<P>
```

Comment:
I'd want to implement `EncodePrivateKey for KeyPair` but I couldn't find a way to recover the seed from a signing key.

Reply:
Because seed -> privatekey is a one way function. We'd have to keep the seed in the keypair if we wanted to do that.

Reply:
https://github.com/RustCrypto/signatures/pull/892